aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -120,9 +120,18 @@ class QLinearInt8 final : public torch::OperatorKernel {
         /*bias=*/bias_ptr,
         /*nCol=*/N);
 
+    // The resulting matrix here is 2-D, let's view it with the original
+    // left hand dimensions of the input. Here are two examples:
+    // 1. If the input tensor is {M, K}, the output tensor is {M, N}.
+    // 2. If the input tensor is {b, M, K}, the output tensor is {b, M, N}.
+    std::vector<int64_t> out_sizes = input.sizes().vec();
+    TORCH_CHECK(
```

Comment:
nit: this is redundant check - you already have a check on input rank above

Reply:
Will remove this.